lib/reqTemplate.js

```diff
@@ -0,0 +1,132 @@
+"use strict";
+
+var URI = require('swagger-router').URI;
+var rbUtil = require('./rbUtil');
+var TAssembly = require('tassembly');
+
+/**
+ * Creates a function that sets a value at the path and initializes all objects along the path.
+ *
+ * @param path the path within an object separated with dots {e.g. test.body.a.b.c}
+ * @returns {Function} the function to set a value at path and initialize object along the path if needed
+ * @private
+ */
+function _setAtPath(path) {
```

Comment:
The function assumes `Object`s. What about arrays? Should we complicate the code to include them as well? Does it make sense to cover something like:

```yaml
body:
  - {a.b}
  - {a.c}
```